src/common/types/annotations.js

```diff
@@ -47,7 +47,8 @@ export type Annotation = {
     modified_at: string,
     modified_by: User,
     permissions: AnnotationPermission,
-    replies?: Array<Reply>,
+    replies?: Array<Comment>,
+    repliesTotal?: number,
```

Comment:
Nearly all V2 API fields use snake_case naming (see the other fields on this object), so this would be `replies_total`.